keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -189,6 +187,9 @@ def __init__(
                 f"`label_encoder.box_variance={label_encoder.box_variance}`."
             )
 
+    def make_predict_function(self, force=False):
```

Comment:
This LGTM if we _are_ planning to make .predict return decoded boxes, but I was under the impression we might be changing this?

Reply:
Yeah that's the consensus -- we still like `predict` to return decoded boxes, but don't like doing so with OD Base Model. So here's the workaround